optuna/integration/mlflow.py

```diff
@@ -117,23 +116,34 @@ def objective(trial):
             it will be broadcasted to each objective with a number suffix in order
             returned by objective function e.g. two objectives and default metric name
             will be logged as ``value_0`` and ``value_1``.
-        nest_trials:
-            Flag indicating whether or not trials should be logged as
-            nested runs. This is often helpful for aggregating trials
-            to a particular study, under a given experiment.
-        tag_study_user_attrs:
-            Flag indicating whether or not to add the study's user attrs
-            to the mlflow trial as tags. Please note that when this flag is
-            set, key value pairs in :attr:`~optuna.study.Study.user_attrs`
-            will supersede existing tags.
+        create_experiment:
+            When ``True``, new MLflow experiment will be created for each optimization run,
```

Comment:
[nit] For sphinx based documentation,

```suggestion
            When :obj:`True`, new MLflow experiment will be created for each optimization run,
```

Reply:
Great catch, thanks!